src/lib.rs

```diff
@@ -11,3 +11,11 @@ pub use stable_hasher::StableHasher;
 
 #[doc(inline)]
 pub use stable_hasher::StableHasherResult;
+
+#[doc(inline)]
+pub use stable_hasher::ExtendedHasher;
+
+pub use sip128::SipHasher128; // TODO: Should SipHasher128 be exposed?
```

Comment:
Should we expose the currently internal `SipHasher128` struct?

IMO I don't think it's necessary and the usefulness it not clear, but on the other-hand it might be weird to have the type-alias where `SipHasher128` is not accessible.

Reply:
I'm not sure it's even possible to keep it private. I think it's OK to have it be public.

Reply:
Alright, it's now properly exposed in a newly created `hashers` module.

> I'm not sure it's even possible to keep it private.

Btw, I tested this with a dummy crate importing this crate and it seems to work fine.

Reply:
Doesn't that lead to a warning like here?
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=43dd52b02997899aee7c802b6c4c9ca5

Reply:
I don't remember having one. After testing a bit I think that was due to `SipHasher128` being `pub` (and not `pub(crate)`) and not using the effective visibility.